server/routes_test.go

```diff
@@ -126,6 +127,25 @@ func Test_Routes(t *testing.T) {
 				assert.Equal(t, "test-model:latest", modelList.Models[0].Name)
 			},
 		},
+		{
+			Name:   "List Model Handler OpenAI",
```

Comment:
Let's add a few more tests on things where we've seen some issues, such as an empty list etc